### ISSUE TITLE
User feedback

### ISSUE THREAD

**Issue by n-scope** [MasterSoft24/CloudCross#22]
Thanks a lot for this amazing piece of software. Great work !

I am not sure if it's the right place, but here are some user feedback:

- To compile on ubuntu-debian based platform, I think the following command should be replaced: 
`qmake-qt5 ../CloudCross.pro ` --> `qmake ../CloudCross.pro `

- Authentication does not seem to work (dropbox or google) via Chrome, perhaps some "magic" performed by the address bar ? I get these kind of errors: 
`Error transferring https://www.googleapis.com/oauth2/v4/token?client_id=xxxx: Bad Request`
Switching to firefox it works. (I use a freshly cloned repo)

- Perhaps you should promote yourself a little better, it is a great project (opensource, cross-platform, support multiple providers, support include/exclude files with wildcard) but I guess I tested every other alternative (insync, gdrive2, overGrive) before even knowing about CloudCross !

**Comment by MasterSoft24** [comment]
Many thanks for your feedback. The situation with Chrome doesn't clear for me. I'm can't reproduce this behavior - on my Arch Linux Chromium doesn't start (at this moment). But on all Chromium-based browsers, all works fine.

And what do you mean when you saying: you should promote yourself a little better?

One more time thanks for your feedback

**Comment by quixotique** [comment]
I think @n-scope means that people searching for cloud storage sync solutions for Linux ought to find CloudCross more easily -- it should appear higher in the list of results.  I am interested in Google Drive sync, and it wasn't mentioned in the first three articles I found, which is a shame.

**Comment by J3rome** [comment]
I am getting the same behaviour as @n-scope concerning the bad request error on Chrome (didn't try on chromium)

I tried the version from apt-get and compilling the master branch. Getting the same issue.

I am running kubuntu 16.04 with qt 5.6.1

I was about to give up and I stumbled on this issue. 
Opening the link in Firefox solved the problem for me.

This is probably the same problem as in #26 & #21
